File: Springboot Code/AgroRent/src/main/java/com/majorproj/agrorent/dto/PaymentDetailsDto.java
package com.majorproj.agrorent.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PaymentDetailsDto {
	Long bookingId;
	String renter;
	String equipmentName;
	Double amount;
	String paymentId;
	String orderId;
}

File: Dot Net Code/AgroRent/DTOs/PaymentDetailsDto.cs
namespace AgroRent.DTOs
{
    public class PaymentDetailsDto
    {
        public int Id { get; set; }
        public string? PaymentId { get; set; }
        public string? OrderId { get; set; }
        public string Status { get; set; } = string.Empty;
        public double Amount { get; set; }
        public DateTime Timestamp { get; set; }
        public int BookingId { get; set; }
    }
}
